src/node_dotenv.cc

```diff
@@ -105,15 +105,22 @@ Local<Object> Dotenv::ToObject(Environment* env) const {
   return result;
 }
 
+// Removes space characters (spaces, tabs and newlines) from
+// the start and end of a given input string
 std::string_view trim_spaces(std::string_view input) {
   if (input.empty()) return "";
-  if (input.front() == ' ') {
-    input.remove_prefix(input.find_first_not_of(' '));
+
+  auto pos_start = input.find_first_not_of(" \t\n");
```

Comment:
should this also include `\r` and `\f`?

Reply:
sounds good to me 🙂, but I'm not sure how to test that, would you be ok with addition without dedicated tests for it?